docs/network-flow-visibility.md

```diff
@@ -88,7 +88,7 @@ not to clone the repository, you can mannually deploy it. The first step is to
 install the ClickHouse Operator, which creates, configures and manages ClickHouse
 clusters. Check the [homepage](https://github.com/Altinity/clickhouse-operator)
 for more information about the ClickHouse Operator. Current checked-in yaml is
-based on their [v0.18.2](https://github.com/Altinity/clickhouse-operator/blob/refs/tags/0.18.2/deploy/operator/clickhouse-operator-install-bundle.yaml)
+based on their [v0.21.0](https://github.com/Altinity/clickhouse-operator/blob/release-0.21.0/deploy/operator/clickhouse-operator-install-bundle.yaml)
```

Comment:
Shall we mention we upgrade from v0.18.2 to v0.21.0 in Theia 0.7?

Reply:
done